exdb/forms.py

```diff
@@ -1,11 +1,37 @@
 from django import forms
 from django.core.exceptions import ValidationError
 from django.utils.timezone import now
+from django.utils.encoding import force_text
+from django.utils.safestring import mark_safe
+from django.utils.html import format_html
 from django.utils.translation import ugettext as _
 from django.forms import ModelForm
 from exdb.models import Experience, ExperienceComment
 
 
+class TypeSelect(forms.Select):
+
+    def render_option(self, selected_choices, option_value, option_label):
+        if option_value is None:
+            option_value = ''  # pragma: no cover
+        option_value = force_text(option_value)
+        if option_value in selected_choices:
+            selected_html = mark_safe(' selected="selected"')
+            if not self.allow_multiple_selected:
+                selected_choices.remove(option_value)
+        else:
+            selected_html = ''
+        css_class = ''
+        choice_dict = {str(c.pk): c for c in self.choices.queryset}
+        if option_value in choice_dict.keys() and not choice_dict[option_value].needs_verification:
```

Comment:
There's no need to use the `.keys()` method when checking for `option_value`'s membership in the dict.